src/vscode-bicep/package.json

```diff
@@ -96,44 +99,119 @@
         "scopeName": "source.bicep",
         "path": "./bicep.tmlanguage"
       }
-    ]
+    ],
+    "commands": [
+      {
+        "command": "bicep.showVisualizer",
+        "title": "Open Visualizer",
+        "category": "bicep",
+        "icon": "$(type-hierarchy-sub)"
+      },
+      {
+        "command": "bicep.showVisualizerToSide",
+        "title": "Open Visualizer to the Side",
+        "category": "bicep",
+        "icon": "$(type-hierarchy-sub)"
+      },
+      {
+        "command": "bicep.showSource",
+        "title": "Open Source",
+        "category": "bicep",
+        "icon": "$(go-to-file)"
+      }
+    ],
+    "menus": {
+      "editor/title": [
+        {
+          "command": "bicep.showVisualizerToSide",
+          "when": "editorLangId == bicep",
+          "alt": "bicep.showVisualizer",
+          "group": "navigation"
+        },
+        {
+          "command": "bicep.showSource",
+          "when": "bicepVisualizerFocus",
+          "group": "navigation"
+        }
+      ],
+      "explorer/context": [
+        {
+          "command": "bicep.showVisualizer",
+          "when": "resourceLangId == bicep",
+          "group": "navigation"
+        }
+      ],
+      "editor/title/context": [
+        {
+          "command": "bicep.showVisualizer",
+          "when": "resourceLangId == markdown",
+          "group": "1_open"
+        }
+      ],
+      "commandPalette": [
+        {
+          "command": "bicep.showVisualizer",
+          "when": "editorLangId == bicep",
+          "group": "navigation"
+        },
+        {
+          "command": "bicep.showVisualizerToSide",
+          "when": "editorLangId == bicep",
+          "group": "navigation"
+        },
+        {
+          "command": "bicep.showSource",
+          "when": "bicepVisualizerFocus",
+          "group": "navigation"
+        }
+      ]
+    }
   },
   "scripts": {
-    "vscode:prepublish": "webpack --mode production",
-    "compile": "tsc -p ./ && npm run copy:tmlanguage",
+    "vscode:prepublish": "npm run build:prod",
+    "build": "webpack --mode development",
+    "build:prod": "webpack --mode production",
+    "watch": "webpack --watch --mode development --stats=minimal",
     "copy:tmlanguage": "ncp ../textmate/bicep.tmlanguage ./bicep.tmlanguage",
-    "watch": "tsc -watch -p ./",
     "lint": "eslint src --ext ts",
     "lint:fix": "eslint src --ext ts --fix",
+    "compile": "tsc && npm run copy:tmlanguage",
```

Comment:
The compile script is only kept for the e2e tests. I just realized that our e2e tests are not verifying the bundled extension script since we started to use webpack. I'm going to update the test runner to launch the bundled extension and clean up the compile script in a separate PR.

Reply:
Btw, do we need any build changes (GH or ADO)?

Reply:
Nope.

Reply:
I ended up updating our e2e tests to verify the bundled extension script and adding snapshot tests for the React components, so I made some changes to the build pipelines.